platform/engineering/collab-cycle/architecture-intent-template.md

```diff
@@ -0,0 +1,79 @@
+# Architecture Intent Template
+
+__*** DRAFT *** DRAFT *** DRAFT ***__
+
+The Architecture Intent meeting helps your team build a solution that meets Veteran-facing Service Platform (VSP) engineering standards and lowers the potential for launch-blocking issues later in the development cycle.
+
+## What is the purpose of Architecture Intent meeting?
+
+The Architecture Intent meeting is less a formal presentation and more of a discussion.  It not only provides OCTO-DE and Platform with an early understanding of the product/feature your team wants to build, but is also an opportunity to collaborate and provide feedback on the intended implementation and surface any adjustments needed to meet VSP engineering standards. The focus is on making sure your code meets user needs within the constraints of the platform your building on.
+
+## When to schedule an Architecture Intent meeting?
+
+You should schedule an Architecture Intent meeting if any of these apply:
+
+- You have questions about one or more points on the template below.
+- You're having trouble locating or getting technical info from other stakeholders or system owners.
+- You're using an architecture pattern not currently found on the va.gov platform.
+- You plan to use a new technology, library or dependency.
+- You're integrating with a new system or API, inside or outside of the VA.
+- You need to coordinate your change with other teams or your change will add work or responsiblity to other teams.
+- You're gathering new [PII]/[PHI] or saving PII/PHI in a new place.
+- There are cost, performance or security implications to your change.
+
+[PII]: https://en.wikipedia.org/wiki/Personal_data
+[PHI]: https://en.wikipedia.org/wiki/Protected_health_information
+
+## Preparing for the Architecture Intent meeting
+
+You should write up your Architecture Intent using the template below.  The Governance Team will need at least **2 business days** to review your materials.
+
+## Architecture Intent Template
```

Comment:
Might be nice if this was broken off into separate a copy/pastable(sp?) document that's linked to from here along with an example for folks to build off.

Reply:
Agreed.  Added a todo and will break this out in the next draft.